test/recipes/80-test_cmp_http.t

```diff
@@ -130,7 +130,7 @@ sub test_cmp_http {
     my $params = shift;
     my $expected_exit = shift;
     my $path_app = bldtop_dir($app);
-    $params = [ '-server', "127.0.0.1:$server_port", @$params ]
+    $params = [ '-server', "127.0.0.1:".$server_port, @$params ]
```

Comment:
This change should not make any difference...

Reply:
Indeed it should not, but during the tests I did this morning I found that in case `$server_port` is still  `"error"`, for whatever reason, the output became `"127.0.0.1:$server_port"` rather than `"127.0.0.1:error". When adapted as given here, it works all fine.

Reply:
What I also found pretty weird is that when `$server_port` was unset, `"127.0.0.1:$server_port"` evaluated to `"127.0.0.1:0"`, so apparently the usage of the variable at this point was taken as scalar context.

Reply:
The number-like string 127.0.0.1 must have confused Perl 🤣

Reply:
What you're saying here makes about zero sense...

Reply:
If you end up getting `"127.0.0.1:$server_port"`, it means that the value of `$server_port` is its own name.
Exactly how that happens, I do not know.

Reply:
Hmm, I cannot reproduce that strange behavior, so reverted to the original, simpler version.

I meanwhile agree that my explanation with scalar context was wrong because
I found that the `"0"` default value does not come from Perl, but from `Mock/test.cnf`.
On this occasion slightly improved the mechanism of specifying and detecting the port number.